@orb.yml

```diff
@@ -0,0 +1,7 @@
+version: 2.1
+
+description: "Common CircleCI tasks for the PHP programming language.
```

Comment:
This will need a multiline indicator such as > 
example: https://github.com/CircleCI-Public/slack-orb/blob/staging/src/%40orb.yml#L3

Reply:
Done in  https://github.com/CircleCI-Public/php-orb/pull/11/commits/c9524c1567cba9ec3398face883ef4799c0396e5